src/client/components/WidgetContainer/widgets/EditoriallySelectedReviewsWidget/EditoriallySelectedReviews.widget.component.js

```diff
@@ -54,54 +54,82 @@ export class EditoriallySelectedReviewsWidget extends AbstractWidget {
+                <img src={coverUrl} />
               </div>
-            <h4 className="editorial-reviews--review--work-title">
-              {work.dcTitle}
-            </h4>
-
-            <div className="editorial-reviews--review--rating-container">
-              <Rating pid={review.pid} rating={review.rating}/>
             </div>
 
-            <div className="editorial-reviews--review--content">
-              "<span dangerouslySetInnerHTML={{__html: review.html}}/>"
-            </div>
-            </div>
-            <div className="editorial-reviews--read-button">
-              <a href={`/anmeldelse/${review.id}`}>
-                Læs anmeldelsen
-              </a>
+            <div className="editorial-reviews--review--right">
+              <div className="main-content">
+                <div className="widget-element--author">
+                  Af:
+                  <a dangerouslySetInnerHTML={{__html: review.owner.displayName}} href={`/profil/${review.owner.id}`} />
+                </div>
+                <h4 className="editorial-reviews--review--work-title">
+                  {work.dcTitle}
+                </h4>
+
+                <div className="editorial-reviews--review--rating-container">
+                  <Rating pid={review.pid} rating={review.rating} />
+                </div>
+
+                <div className="editorial-reviews--review--content">
+                  "<span dangerouslySetInnerHTML={{__html: review.html}} />"
+                </div>
+              </div>
+              <div className="editorial-reviews--read-button">
+                <a href={`/anmeldelse/${review.id}`}>
+                  Læs anmeldelsen
+                </a>
+              </div>
             </div>
-          </div>
 
-        </div>
+          </div>
         </div>
       );
     }
 
     return null;
   }
 
+  getSpinner() {
+    return (
+      <div className="editorially-selected-reviews-widget--spinner-container">
+        <Icon glyph={spinnerSvg} height={150} width={150} />
+      </div>
+    );
+  }
+
+  getShowMoreButton() {
+    return (
+      <div className="editorially-selected-reviews-widget--show-more-button">
+        <a
+          onClick={() => this.setState({expanded: !this.state.expanded})}>
+          <Icon glyph={this.state.expanded ? minusSvg : plusSvg} />
+          {this.state.expanded ? ' VIS FÆRRE' : ' VIS FLERE'}
+        </a>
+      </div>
+    );
+  }
+
   render() {
     const reviews = this.props.widgetReducerProp.reviews;
     const reviewIds = this.props.widgetConfig.reviewIds;
     const works = this.props.widgetReducerProp.works;
-    const showMoreButton = Object.keys(reviews).length > 0 && Object.keys(works).length > 0;
+
+    const showMoreButton = Object.keys(reviews).length > 2 ?
+      this.getShowMoreButton() :
+      null;
+
+    const spinner = this.props.widgetReducerProp.isLoading || this.props.widgetReducerProp.isLoading ?
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct one of the identical sub-expressions on both sides of operator "||" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1764)